attachment_synchronize/tests/common.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+
+from odoo.tests.common import TransactionCase
+
+
+class SyncCommon(TransactionCase):
```

Comment:
Please refactor to SavepointCase and move to setUpClass what does not need to be in setUp.

Reply:
Ah yes of course! Thanks.